db/migrations/20200910165134_update_initial.js

```diff
@@ -0,0 +1,25 @@
+exports.up = function (knex) {
+  return knex.schema
+  // knex.raw('ALTER TABLE "' + table + '" ALTER COLUMN "' + column + '" TYPE ' + newType + '')
+    .raw('ALTER TABLE stories ALTER COLUMN prompt TYPE integer USING (prompt::integer)')
```

Comment:
Unfortunately, if you change around the earlier migration file, this one will now fail because line 4 is doing the change that I suggested in the initial file. @GreysonElkins what order did you write these in? Did you modify the `initial` migration file after it was passing successfully?